README.md

```diff
@@ -75,157 +79,205 @@ W3C Verifiable Credentials specification defines two kinds of objects – Verifi
 
 This specification defines the JWT parameters to pass VPs and VCs signed as JWTs or using Linked Data Proofs.
 
-This specifications also introduces a new artifact VP Token and VC Token to provide VPs and VCs to RPs:
-
-### JWT parameters extention
+This specifications also introduces a new artifact VP Token and VC Token to provide VPs and VCs to RPs.
 
-This specification introduces the following JWT parameters:
+All representations share the same container format.
 
-- vc_jwt:  A claim whose value is a W3C Verifiable Credential object using the JWT representation, which is a JSON string.  The claim’s value may also be an array of W3C Verifiable Credential objects using the JWT representation if the use case calls for multiple JWT VCs.
+## Container Format
 
-- vp_jwt:  A claim whose value is a W3C Verifiable Presentation object using the JWT representation, which is a JSON string.  The claim’s value may also be an array of W3C Verifiable Presentation objects using the JWT representation if the use case calls for multiple JWT VPs.
+### Verifiable Credential
 
-- vc_ldp:  A claim whose value is a W3C Verifiable Credential object using the JSON-LD representation, which is a JSON object.  The claim’s value may also be an array of W3C Verifiable Credential objects using the JSON-LD representation if the use case calls for multiple JSON-LD VCs.
+A verifiable credential container is an array of objects, each of them containing the following fields:
 
-- vp_ldp:  A claim whose value is a W3C Verifiable Presentation object using the JSON-LD representation, which is a JSON object.  The claim’s value may also be an array of W3C Verifiable Presentation objects using the JSON-LD representation if the use case calls for multiple JSON-LD VPs.
+`format`: REQUIRED A JSON string denoting the proof format the credential was returned in. This specification introduces the values `w3cvc-jsonld` and `w3cvc-jwt` to denote credentials in JSON-LD and JWT format, respectively.  
 
-These claims can be added to ID Tokens, Userinfo responses as well as Access Tokens and Introspection response. They MAY also be included as aggregated or distributed claims (see Section 5.6.2 of the OpenID Connect specification [OpenID]).
+`credential` : REQUIRED. A cryptographically verifiable proof in the defined proof format. 
```

Comment:
```suggestion
`credential` : REQUIRED. A W3C Verifiable Credential with a cryptographically verifiable proof in the defined proof format.
```

Reply:
The question being if we want to limit to W3C VCs.. we have talked about support for broader `credentials`s that perhaps do not comply with the W3C spec too.